src/update_available.html

```diff
@@ -26,9 +26,9 @@
             document.write(updateInfo.version)
           </script>
           is the latest
-          <script>document.write(getReleaseChannel())</script>
+          <script>document.write(getUpdatesChannel())</script>
```

Comment:
Can we call workstation.get* directly instead of having to wrap it in the corresponding dialog js files? Looks we had to do that in three places.

Reply:
The problem I ran into is in with import ordering... workstation was brought in multiple places so I was getting workstation already defined errors.

Reply:
I have seen this before. I think possibly we need to change all our requires from `const` to `var`. Or use `import` instead of `require` although I don't know the pros/cons of that